### PR TITLE
Remove windows tray and installer

### PR DESCRIPTION
**Pull request by ascjones**
Part of #8649. Maybe more to remove, creating this PR for now to test the gitlab CI build.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -173,7 +173,7 @@ windows:
     - stable
     - triggers
   script:
-    - sh scripts/gitlab-build.sh x86_64-pc-windows-msvc x86_64-pc-windows-msvc installer "" "" windows
+    - sh scripts/gitlab-build.sh x86_64-pc-windows-msvc x86_64-pc-windows-msvc "" "" "" windows
```
Comment (ascjones) [inline .gitlab-ci.yml]: Not sure what (if anything) should be for the $ARC param instead of installer

File: scripts/gitlab-build.sh [hunk 2]
```diff
@@ -166,19 +165,6 @@ make_pkg () {
...
-  cd nsis
-  curl -sL --url "https://github.com/paritytech/win-build/raw/master/vc_redist.x64.exe" -o vc_redist.x64.exe
-  echo "makensis.exe installer.nsi" > nsis.cmd
-  ./nsis.cmd
-  cd ..
-  cp nsis/installer.exe "parity_"$VER"_"$IDENT"_"$ARC"."$EXT
-  ./sign.cmd $keyfile $certpass "parity_"$VER"_"$IDENT"_"$ARC"."$EXT
-  $MD5_BIN "parity_"$VER"_"$IDENT"_"$ARC"."$EXT -p %h > "parity_"$VER"_"$IDENT"_"$ARC"."$EXT".md5"
-  $SHA256_BIN "parity_"$VER"_"$IDENT"_"$ARC"."$EXT -p %h > "parity_"$VER"_"$IDENT"_"$ARC"."$EXT".sha256"
-}
```
Comment (ascjones) [inline scripts/gitlab-build.sh]: AFAICT this is just installer/tray stuff so can be removed entirely.

**Comment by ascjones** [comment]
@5chdn @General-Beck am I on the right track here?

File: .gitlab-ci.yml (hunk 1 above)
Comment (5chdn) [reply .gitlab-ci.yml]: $ARC is just a string that goes into the binary name. I'm pretty sure it's unused after this PR, however, for consistency reasons, could you change it to `amd64`?

**Comment by ascjones** [comment]
Hm not sure why, no rust stuff modified or anything before the build step

**Comment by 5chdn** [comment]
OK

    * [new tag]             v1.12.0-ci2 -> v1.12.0-ci2

https://gitlab.parity.io/parity/parity/pipelines/20532

**Comment by 5chdn** [comment]
* [new tag]             v1.12.0-ci3 -> v1.12.0-ci3

https://gitlab.parity.io/parity/parity/pipelines/20560

**Comment by ascjones** [comment]
Test fails now:

`test v1::tests::mocked::signing::should_check_status_of_request_when_its_resolved ... FAILED`

Works when running locally... @5chdn can you trigger this again?

**Comment by 5chdn** [comment]
Yeah, hold on. https://gitlab.parity.io/parity/parity/-/jobs/89357

**Comment by 5chdn** [comment]
Looks good!

**Pull request merged**